homeWork7/task1.js

```diff
@@ -0,0 +1,12 @@
+const numbers = [2, -5, 0, 7, -3, 0, 10, -8]
+let positiveCount = 0
+let negativeCount = 0
+let zeroCount = 0
+for (let i of numbers ){
+    if (i > 0) {positiveCount ++} 
```

Comment:
in this case you either need to add `continue` in each `if` block or use` else if`

why? because even if number is positive you still do perform other checks but we know they never be truthy